### PR TITLE
feat: director can return transfer to analyst

### PR DESCRIPTION
**Pull request by kevin-hashimoto**
#316 

- refactor: consolidated button styling config
- feat: Director role IDIR users can return transfer to analyst

### CODE REVIEW

**Review by AlexZorkin — COMMENTED** [review]
Need to fix a few conflicts, but otherwise looks good.

File: frontend/src/views/Transfers/ViewTransfer/ViewTransfer.jsx [hunk 1]
```diff
@@ -224,7 +236,31 @@ export const ViewTransfer = () => {
...
-        ...saveDraftButton(t('saveBtn')),
+        ...redOutlinedButton(t('transfer:refuseTransferBtn'))
+      },
+      {
+        ...outlinedButton(t('transfer:returnToAnalystBtn')),
+        handler: () =>
+          setModalData({
+            primaryButtonAction: () =>
+              updateTransfer({
+                newStatus: 4,
```
Comment (AlexZorkin) [inline frontend/src/views/Transfers/ViewTransfer/ViewTransfer.jsx:247]: Can you think of a way so we can use status names rather than ids here?

**Review by AlexZorkin — APPROVED** [review]
Looks good, just one comment on a possible improvement. We can move the result to a new ticket if its more than a 1.

**Comment by AlexZorkin** [comment]
Just a bump to review the conflicts and comment.

File: frontend/src/views/Transfers/ViewTransfer/ViewTransfer.jsx (hunk 1 above)
Comment (kevin-hashimoto) [reply frontend/src/views/Transfers/ViewTransfer/ViewTransfer.jsx:247]: would the goal be to make it more readable or to pass the actual string value?

few options come to mind:

1.
```
export const statuses = {
  ...
  sent: {
    id: 4,
    label: 'Sent'
  }
  ...
}
```
and then pass in `statuses.sent.id`.

2. create a util function that maps a string value to a status id
3. using the actual string value of a status and handling the mapping on the backend

Comment (AlexZorkin) [reply frontend/src/views/Transfers/ViewTransfer/ViewTransfer.jsx:247]: I think we should use the string value, and then map that to the Enum value on the backend. Updating a transfer on the backend has the same issue, checking against the ids without knowing their value. I will create a new ticket for this refactor. Please just fix the existing conflicts and we'll close this pr.

**Pull request merged**